.github/workflows/_run.yml

```diff
@@ -252,6 +253,7 @@ jobs:
         GITHUB_TOKEN: ${{ steps.checkout.outputs.token && steps.checkout.outputs.token || secrets.GITHUB_TOKEN }}
         ENVOY_DOCKER_BUILD_DIR: ${{ runner.temp }}
         ENVOY_RBE: ${{ inputs.rbe != 'false' && 1 || '' }}
+        GCP_TEST_PROJECT_API_KEY: ${{ inputs.trusted && secrets.gcp-traffic-key || '' }}
```

Comment:
the condition here should not be strictly necessary - this secret should only appear in the traffic ci which is always trusted - but its added as a failsafe - it just means the traffic ci must pass `trusted: true` when calling